docs/tutorials.md

```diff
@@ -103,8 +106,9 @@ The next object describes how this step has to be considered complete. It's a JS
 
 So far we support only one operation: 'signal'. This operation requires the following properties:
 
-| Property    | Description                                                                                                        | Type   | Required                    |
-| ----------- | ------------------------------------------------------------------------------------------------------------------ | ------ | --------------------------- |
-| qml_emitter | A QML object name that has to emit a signal                                                                        | String | Yes if vpn_emitter is empty |
-| vml_emitter | An internal VPN component that has to emit a signal. So far, we support only: **settingHolder** and **controller** | String | Yes if qml_emitter is empty |
-| signal      | The signal name                                                                                                    | String | Yes                         |
+| Property      | Description                                                                                                        | Type   | Required                    |
+| ------------- | ------------------------------------------------------------------------------------------------------------------ | ------ | --------------------------- |
+| query_emitter | A qml-path query to identify the object that has to emit a signal                                                  | String | Yes if vpn_emitter is empty |
+| qml_emitter   | A QML object name that has to emit a signal (pre v2.13 only)                                                       | String | No (use only in < v2.13)    |
+| vml_emitter   | An internal VPN component that has to emit a signal. So far, we support only: **settingHolder** and **controller** | String | Yes if qml_emitter is empty |
```

Comment:
Question: Is this used at all?

Reply:
Yes we do it in tutorials. See: https://searchfox.org/mozilla-vpn-client/search?q=vpn_emitter&path=&case=false&regexp=false